lib/site_template/_config.yml

```diff
@@ -20,7 +20,8 @@ description: > # this means to ignore newlines until "baseurl:"
   line in _config.yml. It will appear in your document head meta (for
   Google search results) and in your feed.xml site description.
 baseurl: "" # the subpath of your site, e.g. /blog
-url: "http://example.com" # the base hostname & protocol for your site
+url: "" # the base hostname & protocol for your site, e.g. http://example.com
+        # [Required] for production environment.
```

Comment:
My vote is to just remove [line 24](https://github.com/ashmaroli/jekyll/blob/debd1b7beaa0307a7b00c70f53c844306a59517c/lib/site_template/_config.yml#L24) and leave the comment on 23.